File: euler/01/1.go
package main

import (
    "fmt"
    "strconv"
)


func main(){
    upper := 1000
    sum := 0
    for i:=0; i<upper; i++{
        if i % 3 == 0 || i % 5 == 0{
            sum += i
        }
    }

    fmt.Println("The sum is " + strconv.Itoa(sum))
}

File: euler/01/1.f90
program euler1
    implicit none

    Integer :: N, i, upper, total=0
    Character(len=20) :: arg
    N = IARGC()
if( N < 1) then
    upper = 1000
else
    call getarg(1,arg) 
    read(arg, *) upper
endif

do i=1,upper-1
        if(mod(i,5)==0 .or. mod(i,3)==0) then
            total = total + i
        endif
enddo

write(*,*) "The sum up to", upper, "is", total

end program euler1
